CONTRIBUTING.md

```diff
@@ -0,0 +1,216 @@
+
+# How to contribute
+
+Thank you for your interest in contributing to NetPyNE project!. NetPyNE is open source and available at [GitHub repository](https://github.com/Neurosim-lab/netpyne).
+
+[NetPyNE](http://netpyne.org/overview.html) is currently being developed and supported by [Neurosin lab](http://neurosimlab.org/) and accepts contributions in the form of bug reports, fixes, feature additions, and documentation improvements (even just typo corrections). The best way to start contributing is by [opening an issue](https://github.com/Neurosim-lab/netpyne/issues) or [fork](https://help.github.com/en/articles/fork-a-repo) the repository on our GitHub page and make a [Pull Request](https://help.github.com/en/articles/creating-a-pull-request) with your changes.
+
+Users and contributors to NetPyNE are expected to follow our [coding conventions](#coding-conventions).
+
+#### Table Of Contents
+
+- [Overview of contribution process](#overview-of-contribution-process)
+- [Getting started](#getting-started)
+  - [Configuring git](#configuring-git)
+  - [Make a local copy of GitHub repository](#make-a-local-copy-of-github-repository)
+  - [Basic git commands](#basic-git-commands)
+  - [Connecting to GitHub with SSH (optional)](#connecting-to-github-with-ssh-optional)
+- [Opening an issue](#opening-an-issue)
+- [Opening a pull request](#opening-a-pull-request)
+- [Coding conventions](#coding-conventions)
+
+## Overview of contribution process
+> _Hint:_ **Working on your first pull request?** Learn how from this guide in GitHub. [How to Contribute to Open Source Project](https://opensource.guide/how-to-contribute/)
+
+In general you’ll be working with three different copies of the Neurosim-lab/netpyne repository codebase: the official remote copy at https://github.com/Neurosim-lab/netpyne (usually called upstream), your remote [fork](https://help.github.com/en/articles/fork-a-repo) of the upstream repository (similar URL, but with your username in place of Nuerosim-lab, and usually called origin), and the local copy of the codebase on your computer. The typical contribution process is to:
+
+1. synchronize your local copy with `upstream`

```

Comment:
also need to synchronize you remote fork with upstream?

Reply:
I think that you update your local copy, and you update your fork when you push to your fork (step 3).